src/util/transition/ngbFadingTransition.ts

```diff
@@ -19,3 +20,19 @@ export const ngbToastFadeOutTransition: NgbTransitionStartFn = ({classList}: HTM
 
   return () => { classList.add('hide'); };
 };
+
+export const ngbNavFadeOutTransition: NgbTransitionStartFn = ({classList}) => {
+  classList.remove('show');
+  return () => classList.remove('active');
+};
+
+export const ngbNavFadeInTransition: NgbTransitionStartFn = (element: HTMLElement) => {
+  element.classList.add('active');
+  reflow(element);
```

Comment:
@fbasso I think we might be able to remove this `reflow` after recent changes to the `ngbRunTransition`. Will test this theory after reviews are done

Reply:
nope, we can't